tests/db/migration_tests/dumps/v202303071.sql

```diff
@@ -441,8 +441,8 @@ CREATE TABLE public.unknownparameter (
 --
```

Comment:
Why did the previous dump get changed?

Reply:
I wanted to have an environment in the dump that had `purge_on_delete` enabled, to test the db migration script. No such environment was present in the existing dump.